src/ts/uifactory.ts

```diff
@@ -456,17 +459,26 @@ export namespace UIFactory {
         }),
         new TitleBar({
           components: [
+            new Container({
+              components: [
+                subtitleMenuCaption,
+                audioMenuCaption,
+              ],
+              cssClasses: ['ui-menucaption'],
+            }),
             new Container({
               components: [
                 new MetadataLabel({ content: MetadataLabelContent.Title }),
-                new SettingsToggleButton({
+                 new SettingsToggleButton({
```

Comment:
Please remove this space, it breaks the 2-spaces indentation used in this project

```suggestion
                new SettingsToggleButton({
```